src/es6-import-test.js

```diff
@@ -0,0 +1,3 @@
+import canonicalGlobalNav from '../dist/index';
```

Comment:
I'm not sure if `src` is good place for that? Maybe we should have separate `test` folder for this?